src/bsoncxx/json.hpp

```diff
@@ -34,16 +34,25 @@ BSONCXX_INLINE_NAMESPACE_BEGIN
 extern const bool k_silence_doxygen;
 
 ///
-/// Converts a BSON document to a JSON string.
+/// An enumeration of the types of Extended JSON that the to_extended_json function accepts
```

Comment:
Nit: the function to_extended_json no longer exists. And can you doxygen mention it somehow?